notebooks/community/ml_ops/stage3/get_started_with_bqml_pipeline_components.ipynb

```diff
@@ -1,1199 +1,1225 @@
 {
```

Comment:
<p>I would describe the importer step here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/299/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>done</p>